src/routes/(app)/members/[studentId]/+page.server.ts

```diff
@@ -60,6 +63,32 @@ export const load: PageServerLoad = async ({ locals, params }) => {
     throw error(404, "Member not found");
   }
   const member = memberResult.value;
+  try {
+    return {
+      form: await superValidate(member, memberSchema),
+      pingForm: await superValidate(emptySchema),
+      member,
+      publishedArticles: publishedArticlesResult.value ?? [],
+      ping: user
+        ? await prisma.ping.findFirst({
+            where: {
+              OR: [
+                {
+                  fromMemberId: member.id,
+                  toMemberId: user?.memberId,
+                },
+                {
+                  fromMemberId: user?.memberId,
+                  toMemberId: member.id,
+                },
+              ],
+            },
+          })
+        : null,
+    };
+  } catch (e) {
+    throw error(500, "Could not fetch ping");
+  }
 
   const allMemberDoors = await getCurrentDoorPoliciesForMember(
     prisma,
```

Comment:
Delete unreachable code.

Reply:
Out of place after merge? Moving it to to be included in return